lib/landrush/action/common.rb

```diff
@@ -71,7 +71,7 @@ def read_machine_hostname
           return machine.config.vm.hostname
         end
 
-        "#{Pathname.pwd.basename}.#{config.tld}"
+        "#{Pathname.pwd.basename}.#{config.tld[0]}"
```

Comment:
My understanding is that method sets a default DNS hostname entry. If `config.vm.hostname` is not defined in the Vagrantfile it generates one from the `landrush.tld`. A single default hostname entry is suitable for my use cases so, to keep code changes to a minimum, I just use the first tld.

In my Vagrantfile I use

```
config.landrush.tld = [ 'one.org', 'two.org' ]
config.landrush.host 'one.org'
config.landrush.host 'two.org'
```

So, yes, I use `config.landrush.host` but I only define hosts at the tld level and leverage landrush's wildcard subdomains. This allows me to configure apache/nginx namebased virtual hosts: a.one.org, b.one.org, a.two.org without having to manually manage and reload individual hostnames in the Vagrantfile.